CHANGELOG/v1.12.1.md

```diff
@@ -0,0 +1,27 @@
+## Changes by Kind
+
+### Bug or Regression
+
+- Fix: DNSPrefix error on existing tenant cluster (#4368, @mihaiandreiratoiu)
+
+### Other (Cleanup or Flake)
+
+- Bump CAPI to v1.5.4 (#4344, @mboersma)
+
+### Uncategorized
+
+- Support AKS BYOCNI by allowing networkPlugin: none (#4318, @illrill)
```

Comment:
nit: I see this original PR was classified as `kind/bug`.

Reply:
/hold

Reply:
fixed

/hold cancel